Userland/DevTools/HackStudio/HackStudioWidget.h

```diff
@@ -130,7 +130,11 @@ class HackStudioWidget : public GUI::Widget {
     void create_location_history_actions();
 
     void add_new_editor_tab_widget(GUI::Widget& parent);
-    void add_new_editor(GUI::TabWidget& parent);
+    enum class AndFocus {
```

Comment:
`AndFocus` sounds a bit awkward, no?
It might be me, but I prefer simply `Focus` or `ChangeFocus`.

Reply:
lol, this fix was dumb :sweat_smile: 
Simplified this to a one line change :^)